assets/demos/README.md

```diff
@@ -0,0 +1,97 @@
+# Benchpress Demo Application
+
+* Clone AspNetCore.Docs repository
+
+```powershell
+git clone https://github.com/dotnet/AspNetCore.Docs.git
+```
+
+* Create a new resource group with a unique name
+
+```powershell
+$suffix = (Get-Random).ToString("x8")
+$location = "westus2"
+
+az group create --name "benchpress-rg-${suffix}" --location "${location}"
+```
+
+* Deploy the demo application infrastructure
+
+   * Application Insights
+   * Action Group
+   * App Service Plan
+   * Web App
+
+```powershell
+az deployment group create --resource-group "benchpress-rg-${suffix}" --template-file "main.bicep" --parameters suffix="${suffix}"
+```
+
+* Go to the demo application folder
+
+```powershell
+Push-Location -Path .\AspNetCore.Docs\aspnetcore\mvc\controllers\filters\samples\6.x\FiltersSample
+```
+
+* Deploy the demo application
+
+```powershell
+az webapp up --name "benchpress-web-${suffix}" --resource-group "benchpress-rg-${suffix}" --location "${location}" --sku "F1"
+```
+
+* Go back to the previous directory
+
+```powershell
+Pop-Location
+```
+
+## Running the tests
+
+* Setup the environment variables
+
+```powershell
+# Can be retrieved from the output of the deploy command
+$env:ENVIRONMENT_SUFFIX = "${suffix}"
+```
+
+* Run the tests by calling the test script file, or, `Invoke-Pester` command
```

Comment:
Note the configuration script should only include spinning up the canonical app and not test execution so we can use in the pipeline and have a separate step for test execution

Reply:
👍